sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1884,6 +1889,146 @@ def plot_ild_shear(self,subs=None):
         return plot(self._ild_shear.subs(subs), (x, 0, l),  title='I.L.D. for Shear',
                xlabel=r'$\mathrm{X}$', ylabel=r'$\mathrm{V}$', line_color='blue',show=True)
 
+    def _solve_for_ild_moment(self):
```

Comment:
Instead of having 3 different `_solve_for_ild_....()` functions, we can have a single `_solve_for_ild_equations`, since they more or less perform the same operations.

Reply:
I'm not sure that will work because the function will return both shear eq and moment eq. In the shear function, moment eq is not required, similarly for moment function, shear eq is not required. These will be unused variables and the Travis tests for sympy shows error when you leave something unused.

Reply:
Try using an underscore instead. Like for the case where shear is not required use:

 `_ , moment = _solve_for_ild_equations()`

I think this should work!

Reply:
I have removed the additional functions